verifiers/container/container.go

```diff
@@ -0,0 +1,15 @@
+package container
+
+import (
+	"strings"
+
+	crname "github.com/google/go-containerregistry/pkg/name"
+)
+
+var GetImageDigest = func(image string) (string, error) {
```

Comment:
Note: would this break if we verify in parallel different images that require changing this global variable?
I don't think we're going to need that any time soon.. unless the unit tests have to.

Reply:
Hmmm The purpose of this global is just for testing -- only testing will use the modified vars (and always will)

> I wonder whether we this should be a func GetImageDigest() and have an interface that this function implement in a structure. 

Actually, that can work. We can do a file-based impl and a registry-based implementation of the container. Then we can instantiate based on the params (was image str a local file or a container reference)

Reply:
It's fine to do that in a follow-up PR. Up to you.